website/templates/project/addon/widget.mako

```diff
@@ -9,14 +9,17 @@
 
                 </div>
             </div>
+    % else:
```

Comment:
When I looked at this as a whole file and not in files changed view, I realized that the %else blocks etc could be moved to the left one tab. The first if block starts all the way to the left, and everything else is indented one extra level, so it took me a minute to figure out what was going on. This is also super minor.